source/fundamentals/crud.txt

```diff
@@ -1,13 +1,26 @@
-
-====
-CRUD
-====
+===============
+CRUD Operations
+===============
 
 .. default-domain:: mongodb
 
 .. toctree::
-   :titlesonly:
-   :maxdepth: 1
+   :caption: CRUD Operations
+
+   /fundamentals/crud/read-operations
+   /fundamentals/crud/write-operations
+
+CRUD (Create, Read, Update, Delete) operations allow you to work with
+the data stored in MongoDB.
```

Comment:
I wonder if it might be useful to mention what they impact rather than what they "allow you to [do]".
 E.g.:

You can specify CRUD (Create, Read, Update, Delete) operations to read and modify data in your MongoDB instance.

or (might be too passive):

CRUD (...) operations enable you to read and modify data in MongoDB.

Reply:
I've reworded this and removed some wording that I felt was adding nothing to the page.